solidity/tsconfig.json

```diff
@@ -1,7 +1,8 @@
 {
   "compilerOptions": {
     "downlevelIteration": true,
-    "esModuleInterop": true
+    "esModuleInterop": true,
+    "resolveJsonModule": true
```

Comment:
Why do we need this change? It would be great to align `compilerOptions` in `tsconfig.json` across all projects. I am looking at `keep-core/solidity/ecdsa/tsconfig.json`, `keep-core/solidity/random-beacon/tsconfig.json` and this `tsconfig.json` and we have three versions of `compilerOptions`.

Reply:
It's no longer needed after recent updates. 1b6897a793241926504ca1ae25746df907899f47

Reply:
`compilerOptions` depends on the project. I'm not sure we need to make them the same.